src/app/projects/components/project-settings/project-settings.component.ts

```diff
@@ -554,20 +554,18 @@ export class ProjectSettingsComponent implements OnInit, OnDestroy, AfterViewIni
   }
 
   focusModalInputBox(event: Event, inputBoxName: string) {
-    if (event.target instanceof HTMLInputElement) {
-      const modalDiv = event.target.nextSibling;
-      if (modalDiv instanceof HTMLElement) {
-        const inputChildren = modalDiv.getElementsByTagName('INPUT');
-        for (var i = 0; i < inputChildren.length; ++i) {
-          var node = inputChildren[i];
-          if (
-            node instanceof HTMLElement &&
-            node.getAttribute('name') == inputBoxName
-          ) {
-            node.focus();
-            return;
-          }
-        }
+    event.preventDefault();
+    const inputChildren = document.getElementsByName(inputBoxName);
```

Comment:
get elements by name isn't superfast and an array. get element by id should work better for this to work we need to give it an id (and to prevent autocomplete values autocomplete="off" I think)